Document/0x06e-Testing-Cryptography.md

```diff
@@ -75,18 +68,18 @@ Mobile SDKs offer standard implementations of RNG algorithms that produce number
 
 #### Static Analysis
 
-Apple provides developers with the Randomisation Services application programming interface (API) that generates cryptographically secure random numbers<sup>[1]</sup>.
+Apple provides developers with the Randomisation Services application programming interface (API) that generates [cryptographically secure random numbers](https://developer.apple.com/reference/security/randomization_services "Randomization Services").
```

Comment:
Maybe the link should be around "Randomisation Services" instead of around "cryptographically secure random numbers". Also, "Randomisation" should be with a "z".